__mocks__/popper.js.js

```diff
@@ -0,0 +1,12 @@
+import PopperJs from 'popper.js';
+
+export default class Popper {
+    static placements = PopperJs.placements;
+
+    constructor() {
+        return {
+            destroy: () => { },
+            scheduleUpdate: () => { }
+        };
+    }
+}
```

Comment:
Used to suppress console warnings in the unit tests since popper.js is basically only written to work in a real DOM.